README.md

```diff
@@ -17,16 +17,11 @@ Assuming you're TTS staff, it's recommended that you **make the change in a bran
      1. Add a file for the domain (or subdomain, if the second-level domain isn't being added), to create the [public hosted zone](http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingHostedZone.html).
         - [`analytics.usa.gov`](terraform/analytics.usa.gov.tf) is a good example to copy from.
         - You'll be using Terraform's [`aws_route53_zone`](https://www.terraform.io/docs/providers/aws/d/route53_zone.html).
+        - If it's an existing domain, you'll want to make sure all existing records are copied over, so that they don't break when the cutover happens. You can ask the existing DNS managers for a list of records or a [zone file](https://en.wikipedia.org/wiki/Zone_file) for the domain and all subdomains.
      1. After the pull request is merged, to get the name servers for your domain check the output for your build in [CircleCI](https://circleci.com/gh/18F/dns). If you need further assistance, check with [#admins-dns](https://gsa-tts.slack.com/messages/C4L58EQ5T).
      1. Change the nameservers for the domain to point to AWS.
-        - For `.gov` domains, this will be done by the "domain manager" in [dotgov.gov](https://www.dotgov.gov/). The domain manager is likely someone in the respective agency's IT department.
-1. Add the relevant additional [record sets](http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/rrsets-working-with.html). In Terraform, these are known as [`aws_route53_record`](https://www.terraform.io/docs/providers/aws/r/route53_record.html)s. Generally speaking, the required arguments are:
-   - `zone_id`
-   - `name`
-   - `type`
-   - Either `alias` or `records`
-   - If `alias`, then `evaluate_target_health` is also required and `ttl` is not allowed.
-   - If `records`, then `ttl` is also required and `evaluate_target_health` is not allowed.
```

Comment:
I think we can defer to the canonical documentation around the different arguments.